src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,322 @@
+    UINT32 frameHeight; //!< Only valid for video. Represents the height of the last encoded frame
+    UINT32 frameBitDepth; //!< Only valid for video. Represents the bit depth per pixel of the last encoded frame. Typical values: 24, 30, 36
+    UINT32 nackCount; //!< Count the total number of Negative ACKnowledgement (NACK) packets received by this sender.
+    UINT32 firCount; //!< Only valid for video. Count the total number of Full Intra Request (FIR) packets received by this sender
+    UINT32 pliCount; //!< Only valid for video. Count the total number of Picture Loss Indication (PLI) packets received by this sender
+    UINT32 sliCount; //!< Only valid for video. Count the total number of Slice Loss Indication (SLI) packets received by this sender
+    UINT32 qualityLimitationResolutionChanges; //!< Only valid for video. The number of times that the resolution has changed because we are quality limited
+    INT32 fecPacketsSent; //!< Total number of RTP FEC packets sent for this SSRC. Can also be incremented while sending FEC packets in band
+    UINT64 lastPacketSentTimestamp; //!< The timestamp at which the last packet was sent for this SSRC
+    UINT64 headerBytesSent; //!< Total number of RTP header and padding bytes sent for this SSRC
+    UINT64 bytesDiscardedOnSend; //!< Total number of bytes for this SSRC that have been discarded due to socket errors
+    UINT64 retransmittedPacketsSent; //!< The total number of packets that were retransmitted for this SSRC
+    UINT64 retransmittedBytesSent; //!< The total number of PAYLOAD bytes retransmitted for this SSRC
+    UINT64 targetBitrate; //!< Current target TIAS bitrate configured for this particular SSRC
+    UINT64 totalEncodedBytesTarget; //!< Increased by the target frame size in bytes every time a frame has been encoded
+    UINT64 framesPerSecond; //!< Only valid for video. The number of encoded frames during the last second
+    UINT64 qpSum; //!< Only valid for video. The sum of the QP values of frames encoded by this sender. QP value depends on the codec
+    UINT64 totalSamplesSent; //!< Only valid for audio. The total number of samples that have been sent over this RTP stream
+    UINT64 samplesEncodedWithSilk; //!< Only valid for audio and when the audio codec is Opus. Represnets only SILK portion of codec
+    UINT64 samplesEncodedWithCelt; //!< Only valid for audio and when the audio codec is Opus. Represnets only CELT portion of codec
+    UINT64 totalEncodeTime; //!< Total number of seconds that has been spent encoding the framesEncoded frames of the stream
+    UINT64 totalPacketSendDelay; //!< Total time (seconds) packets have spent buffered locally before being transmitted onto the network
+    UINT64 averageRtcpInterval; //!< The average RTCP interval between two consecutive compound RTCP packets
+    PQualityLimitationDurationsRecord qualityLimitationDurations; //!< Total time (seconds) spent in each reason state
```

Comment:
This is a pointer type. Not sure how this will be used? Is this a fixed size array? How does this get allocated?

Reply:
My bad. Fixed.